cp/longest substring.py

```diff
@@ -0,0 +1,13 @@
+s=str(input())
+k=int(input())
+d={}
+length=0
+if (1 <= len(s) <= 10**4) and (1 <= k <= 26):
```

Comment:
You don't need this if condition, the constraints mean that u can assume that user's input iwll be in this range, so there's no need to check if the input lies in this range